tdh/resource_data_plane.go

```diff
@@ -418,3 +454,21 @@ func saveFromDataPlaneResponse(ctx *context.Context, diagnostics *diag.Diagnosti
 	state.Tags = list
 	return 0
 }
+
+func (r *dataPlaneResource) validateDpCreateRequest(plan *dataPlaneResourceModel) string {
+	if plan.ProviderName.ValueString() == "tas" {
+		if plan.Network.IsNull() || plan.AvailabilityZone.IsNull() {
+			return `Mandatory Fields for Tas data plane creations are missing. Please verify the request body with the schema `
+		} else if plan.Shared.ValueBool() == false {
```

Comment:
what if shared is true, and org_id is passed?

Reply:
We do not use it while composing the request payload